src/dirs.proj

```diff
@@ -17,6 +17,9 @@
     <Project Include="XMakeTasks\UnitTests\Microsoft.Build.Tasks.UnitTests.csproj" />
     <Project Include="XMakeBuildEngine\UnitTests\Microsoft.Build.Engine.UnitTests.csproj" />
     <Project Include="XMakeBuildEngine\UnitTestsPublicOM\Microsoft.Build.Engine.OM.UnitTests.csproj" />
+
+    <Project Include="Test\Projects\Dependency\Dependency.csproj" />
+    <Project Include="Test\Projects\TaskWithDependency\TaskWithDependency.csproj" />
```

Comment:
@cdmihai So the projects under Samples aren't really samples per se, but projects for use in test runs?